web/src/components/Footer.tsx

```diff
@@ -1,36 +1,28 @@
+import Typography from '@mui/material/Typography'
 import { styled } from '@mui/material/styles'
 import React, { ReactElement, ReactNode } from 'react'
 
 import buildConfig from '../constants/buildConfig'
 
 type FooterProps = {
   children: ReactNode[] | ReactNode
-  overlay?: boolean
 }
 
-const FooterContainer = styled('footer')<{ overlay: boolean }>`
+const FooterContainer = styled('footer')`
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
-  padding: ${props => (props.overlay ? '0 10px' : '15px 5px')};
-  margin-top: auto;
-  background-color: ${props =>
-    props.overlay ? `rgba(255, 255, 255, 0.5)` : props.theme.legacy.colors.backgroundAccentColor};
-  box-shadow: 0 2px 3px 3px rgb(0 0 0 / 10%);
-
-  ${props => (props.overlay ? 'color: rgba(0, 0, 0, 0.75);' : '')}
-  & > * {
-    margin: ${props => (props.overlay ? 0 : '5px')};
-    color: ${props => props.theme.isContrastTheme && !props.overlay && props.theme.legacy.colors.textColor};
-  }
-
-  & > *::after {
-    padding-inline-end: 10px;
-    content: '';
-  }
-
-  & > *:last-child::after {
-    content: '';
+  align-items: center;
+  gap: ${props => props.theme.spacing(2)};
+  background-color: ${props => props.theme.legacy.colors.backgroundAccentColor};
+  padding-bottom: ${props => props.theme.spacing(2)};
+  color: ${props => props.theme.palette.text.primary};
+
+  ${props => props.theme.breakpoints.up('md')} {
+    background-color: ${props =>
+      props.theme.isContrastTheme ? props.theme.palette.secondary.dark : props.theme.palette.secondary.light};
```

Comment:
💭 I think we should change this to tertiary as discussed in https://github.com/digitalfabrik/integreat-app/pull/3505#discussion_r2329545803.
```suggestion
    background-color: ${props =>
      props.theme.isContrastTheme ? props.theme.palette.tertiary.dark : props.theme.palette.tertiary.light};
```